projects/gateway2/krtcollections/uniqueclients_test.go

```diff
@@ -139,7 +139,7 @@ func TestUniqueClients(t *testing.T) {
 				g.Eventually(func() []krtcollections.UniqlyConnectedClient {
 					allUcc = ucc.List()
 					return allUcc
-				}, "1s").Should(HaveLen(len(tc.result)))
+				}, "1s").Should(HaveLen(len(tc.result) - (i + 1)))
```

Comment:
nit: a comment on why `i + 1` here would be helpful

Reply:
done